content/en/docs/components/training/job-scheduling.md

```diff
@@ -87,6 +89,23 @@ configure operator to select the scheduler name for gang-scheduling in the follo
 ...
 ```
 
+- mpi-operator
+
+```diff
+...
+    spec:
+      containers:
+      - args:
++       - --gang-scheduling
++       - volcano
+        - -alsologtostderr
+        - --lock-namespace
+        - mpi-operator
+        image: mpioperator/mpi-operator:master
```

Comment:
I think it's better not to put a tag so that it targets latest, or put a released tag.

Reply:
Good point. Let's use the `v0.4.0`. I will fix the tag once we release the mpi-operator v0.4.0.

Reply:
0.4.0

Reply:
Done.

Reply:
or just `0.4`

Reply:
selected `0.4.0`.